src/neo/SmartContract/Native/Designate/DesignateContract.cs

```diff
@@ -40,7 +40,7 @@ public ECPoint[] GetDesignatedByRole(StoreView snapshot, Role role)
             return snapshot.Storages[CreateStorageKey((byte)role)].GetInteroperable<NodeList>().ToArray();
         }
 
-        [ContractMethod(0, CallFlags.AllowModifyStates)]
+        [ContractMethod(0_02000000, CallFlags.AllowModifyStates)]
```

Comment:
This method can be called by the committee only. Why do we charge the committee?

Reply:
I think we should not distinguish who can call it, maybe only if it's `System` but not according to the signers